code/acting/src/acting/vehicle_controller.py

```diff
@@ -11,6 +11,8 @@
 
 PURE_PURSUIT_CONTROLLER: int = 1
 STANLEY_CONTROLLER: int = 2
+STANLEY_CONTROLLER_MIN_V: float = 4.0  # ~14kph
+# STANLEY_CONTROLLER_MAX_V: float = 13.89  # ~50kph
```

Comment:
Does the Stanley controller still work properly beyond 50 km/h?

Reply:
Neither controller was tested beyond ~30kp/h. `STANLEY_CONTROLLER_MAX_V` is a leftover from a time where the switching between controllers was simply handled by if-statements. If i recall correctly last years project used Pure Pursuit for higher speeds.